style.css

```diff
@@ -0,0 +1,72 @@
+html{
+    background-color: #ffdd99;
+}
+
+.header{
+    width: 100%;
+    color: #ffdd99;
+    background-color: #336600;
+    text-align: center;
+}
+.header h1{
+    padding: 10px;
+    font-size: 3.5em;
+}
+
+
+.nav ul{
+    background-color: #336600;
+    text-align: center;
+    margin: 0px;
+    padding: 10px;
+    overflow: hidden;
+}
+
+.nav ul li{
+    display: inline;
+    text-align: center;
+    font-size: 2em;
+    padding: 40px;
+    color: #ffdd99;
+}
+
+.ext{
```

Comment:
Maybe find a more suggestive name for this class. I'm not sure what "ext" means.